packages/extension/src/background/actionHandlers.ts

```diff
@@ -256,3 +277,19 @@ export const handleActionRejection = async (
       assertNever(action)
   }
 }
+
+// TODO: use the function from web3-sdk
```

Comment:
Once [this PR](https://github.com/alephium/alephium-web3/pull/437) is merged, we can use the `TransactionBuilder.buildUnsignedTx` function from the SDK.